src/components/nodes/task/task.vue

```diff
@@ -73,7 +96,7 @@ export default {
   },
   computed: {
     hasTaskMarker() {
-      return this.shape.attr('image/xlink:href') ? true : false;
+      return !!this.shape.attr('image/xlink:href');
```

Comment:
This is a suggested simplification from PHPStorm. Not sure how I feel about this...